packages/main/src/chat/chat-manager.ts

```diff
@@ -40,7 +46,17 @@ export class ChatManager {
     ) => void,
   ) {}
 
-  public init(): void {
+  public async init(): Promise<void> {
+    const directory = new Directories().getChatPersistenceDirectory();
```

Comment:
issue: we should not create a new Directories instance, it needs to be injected through inversify